Traceback (most recent call last):
  File "/home/ryan/miniconda3/envs/nbdev/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/ryan/miniconda3/envs/nbdev/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/ryan/miniconda3/envs/nbdev/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 173, in wrapped
    return loop.run_until_complete(inner)
  File "/home/ryan/miniconda3/envs/nbdev/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/ryan/miniconda3/envs/nbdev/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/ryan/miniconda3/envs/nbdev/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/ryan/miniconda3/envs/nbdev/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
google_data = data[data['symbol'] == 'GOOGL']

plt.plot(google_data['date'], google_data['open'])

# remove data > 2014-03-01

google_data_pre_2014 = google_data[ google_data['date'] < pd.to_datetime('2014-03-01')]
plt.plot(google_data_pre_2014['date'], google_data_pre_2014['open'])
plt.xlabel('date')
plt.ylabel('opening price (\$)');
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[3], line 7[0m
[1;32m      3[0m plt[38;5;241m.[39mplot(google_data[[38;5;124m'[39m[38;5;124mdate[39m[38;5;124m'[39m], google_data[[38;5;124m'[39m[38;5;124mopen[39m[38;5;124m'[39m])
[1;32m      5[0m [38;5;66;03m# remove data > 2014-03-01[39;00m
[0;32m----> 7[0m google_data_pre_2014 [38;5;241m=[39m google_data[ [43mgoogle_data[49m[43m[[49m[38;5;124;43m'[39;49m[38;5;124;43mdate[39;49m[38;5;124;43m'[39;49m[43m][49m[43m [49m[38;5;241;43m<[39;49m[43m [49m[43mpd[49m[38;5;241;43m.[39;49m[43mto_datetime[49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43m2014-03-01[39;49m[38;5;124;43m'[39;49m[43m)[49m]
[1;32m      8[0m plt[38;5;241m.[39mplot(google_data_pre_2014[[38;5;124m'[39m[38;5;124mdate[39m[38;5;124m'[39m], google_data_pre_2014[[38;5;124m'[39m[38;5;124mopen[39m[38;5;124m'[39m])
[1;32m      9[0m plt[38;5;241m.[39mxlabel([38;5;124m'[39m[38;5;124mdate[39m[38;5;124m'[39m)

File [0;32m~/miniconda3/envs/nbdev/lib/python3.10/site-packages/pandas/core/ops/common.py:76[0m, in [0;36m_unpack_zerodim_and_defer.<locals>.new_method[0;34m(self, other)[0m
[1;32m     72[0m             [38;5;28;01mreturn[39;00m [38;5;28mNotImplemented[39m
[1;32m     74[0m other [38;5;241m=[39m item_from_zerodim(other)
[0;32m---> 76[0m [38;5;28;01mreturn[39;00m [43mmethod[49m[43m([49m[38;5;28;43mself[39;49m[43m,[49m[43m [49m[43mother[49m[43m)[49m

File [0;32m~/miniconda3/envs/nbdev/lib/python3.10/site-packages/pandas/core/arraylike.py:48[0m, in [0;36mOpsMixin.__lt__[0;34m(self, other)[0m
[1;32m     46[0m [38;5;129m@unpack_zerodim_and_defer[39m([38;5;124m"[39m[38;5;124m__lt__[39m[38;5;124m"[39m)
[1;32m     47[0m [38;5;28;01mdef[39;00m [38;5;21m__lt__[39m([38;5;28mself[39m, other):
[0;32m---> 48[0m     [38;5;28;01mreturn[39;00m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_cmp_method[49m[43m([49m[43mother[49m[43m,[49m[43m [49m[43moperator[49m[38;5;241;43m.[39;49m[43mlt[49m[43m)[49m

File [0;32m~/miniconda3/envs/nbdev/lib/python3.10/site-packages/pandas/core/series.py:6119[0m, in [0;36mSeries._cmp_method[0;34m(self, other, op)[0m
[1;32m   6116[0m lvalues [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39m_values
[1;32m   6117[0m rvalues [38;5;241m=[39m extract_array(other, extract_numpy[38;5;241m=[39m[38;5;28;01mTrue[39;00m, extract_range[38;5;241m=[39m[38;5;28;01mTrue[39;00m)
[0;32m-> 6119[0m res_values [38;5;241m=[39m [43mops[49m[38;5;241;43m.[39;49m[43mcomparison_op[49m[43m([49m[43mlvalues[49m[43m,[49m[43m [49m[43mrvalues[49m[43m,[49m[43m [49m[43mop[49m[43m)[49m
[1;32m   6121[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39m_construct_result(res_values, name[38;5;241m=[39mres_name)

File [0;32m~/miniconda3/envs/nbdev/lib/python3.10/site-packages/pandas/core/ops/array_ops.py:344[0m, in [0;36mcomparison_op[0;34m(left, right, op)[0m
[1;32m    341[0m     [38;5;28;01mreturn[39;00m invalid_comparison(lvalues, rvalues, op)
[1;32m    343[0m [38;5;28;01melif[39;00m lvalues[38;5;241m.[39mdtype [38;5;241m==[39m [38;5;28mobject[39m [38;5;129;01mor[39;00m [38;5;28misinstance[39m(rvalues, [38;5;28mstr[39m):
[0;32m--> 344[0m     res_values [38;5;241m=[39m [43mcomp_method_OBJECT_ARRAY[49m[43m([49m[43mop[49m[43m,[49m[43m [49m[43mlvalues[49m[43m,[49m[43m [49m[43mrvalues[49m[43m)[49m
[1;32m    346[0m [38;5;28;01melse[39;00m:
[1;32m    347[0m     res_values [38;5;241m=[39m _na_arithmetic_op(lvalues, rvalues, op, is_cmp[38;5;241m=[39m[38;5;28;01mTrue[39;00m)

File [0;32m~/miniconda3/envs/nbdev/lib/python3.10/site-packages/pandas/core/ops/array_ops.py:129[0m, in [0;36mcomp_method_OBJECT_ARRAY[0;34m(op, x, y)[0m
[1;32m    127[0m     result [38;5;241m=[39m libops[38;5;241m.[39mvec_compare(x[38;5;241m.[39mravel(), y[38;5;241m.[39mravel(), op)
[1;32m    128[0m [38;5;28;01melse[39;00m:
[0;32m--> 129[0m     result [38;5;241m=[39m [43mlibops[49m[38;5;241;43m.[39;49m[43mscalar_compare[49m[43m([49m[43mx[49m[38;5;241;43m.[39;49m[43mravel[49m[43m([49m[43m)[49m[43m,[49m[43m [49m[43my[49m[43m,[49m[43m [49m[43mop[49m[43m)[49m
[1;32m    130[0m [38;5;28;01mreturn[39;00m result[38;5;241m.[39mreshape(x[38;5;241m.[39mshape)

File [0;32mops.pyx:107[0m, in [0;36mpandas._libs.ops.scalar_compare[0;34m()[0m

[0;31mTypeError[0m: '<' not supported between instances of 'str' and 'Timestamp'

